src/runtime/ui-slot-composer.ts

```diff
@@ -266,7 +266,13 @@ export class UiSlotComposer {
       if (container) {
         Object.assign(content, {
           containerSlotName: container.targetSlot.name,
-          containerSlotId: container.targetSlot.id
+          containerSlotId: container.targetSlot.id,
+        });
+      }
+      const modality = particle.recipe.modality;
+      if (!modality.all) {
```

Comment:
Shouldn't we send 'all' if the modality is 'all' (instead of sending no modality)?

Reply:
slot composer doesn't know what are all the modalities that uibroker can handle.